ipv8/messaging/interfaces/endpoint.py

```diff
@@ -11,6 +11,8 @@
 except ImportError:
     netifaces = None
 
+# pylint: disable=c-extension-no-member
```

Comment:
Please enable linting instead of disabling it:

```patch
diff --git a/.pylintrc b/.pylintrc
index 383925d..02d8f02 100644
--- a/.pylintrc
+++ b/.pylintrc
@@ -33,7 +33,7 @@ unsafe-load-any-extension=no
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code
-extension-pkg-whitelist=libtorrent
+extension-pkg-whitelist=libtorrent,netifaces
 
 # Allow optimization of some AST trees. This will activate a peephole AST
 # optimizer, which will apply various small optimizations. For instance, it can
```

Reply:
Done

Reply:
Thanks 👍